docs/rum_getting_started.md

```diff
@@ -15,20 +15,26 @@ The Datadog Android SDK supports Android 4.4 (API level 19)+.
 
 ### Declare SDK as dependency
 
-Declare [dd-sdk-android][1] and the [gradle plugin][12] as a dependency in your `build.gradle` file.
+Declare [dd-sdk-android][1] and the [gradle plugin][12] as a dependency in your `build.gradle` file within your `app`directory
 
 ```
+buildscript {
+    dependencies {
+        classpath("com.datadoghq:dd-sdk-android-gradle-plugin:x.x.x")
+    }
+}
 plugins {
     id("dd-sdk-android-gradle-plugin")
```

Comment:
```suggestion
    id("com.datadoghq.dd-sdk-android-gradle-plugin")
```

according to https://plugins.gradle.org/plugin/com.datadoghq.dd-sdk-android-gradle-plugin